qa/suites/fs/cephadm/renamevolume/distro/centos_latest.yaml

```diff
@@ -0,0 +1 @@
+.qa/distros/supported/centos_latest.yaml
```

Comment:
this is not going to work. you'll need to use `0-single-container-host.yaml` as in https://github.com/ceph/ceph/tree/master/qa/suites/rbd/iscsi